### ISSUE TITLE
[Belo Horizonte/MG - Híbrido] Cientista de Dados na BHS

### ISSUE THREAD

**Issue by thothismybro** [datascience-br/vagas#166]
## Descrição da vaga

Projeto em um cliente do segmento de engenharia civil, para atuação no time de Data Analytics.

Suas atividades serão:
· Interpretar todo universo de dados da empresa;
· Criar, treinar e validar modelos preditivos e prescritivos utilizando técnicas de IA tradicional (como machine learning) e IA generativa (como deep learning);
· Interpretar os resultados dos modelos e análises para gerar insights acionáveis que suportem a tomada de decisão estratégica;
· Desenvolvimento de modelos preditivos utilizando técnicas estatísticas e algoritmos de aprendizado de máquina (ex.: regressão logística, gradient boosting e modelos aplicados a Marketing Mix Modeling – MMM);
· Realização de análises exploratórias, testes de hipóteses e estudos analíticos para apoiar iniciativas de risco, marketing e tomada de decisão.
· Participação na descoberta e no desenho de soluções de Marketing Mix Modeling (MMM), incluindo análise de fontes de dados, definição de variáveis de mídia e avaliação de estruturas estatísticas e bayesianas;
· Automação de pipelines de dados e rotinas diárias de scoring em ambiente de nuvem;
· Documentação de metodologias, métricas de validação, processos de monitoramento de desempenho e definições analíticas para iniciativas de modelagem;
· Trabalho em parceria com equipes de negócios, marketing, engenharia e produto.

## Local

Híbrido/BH (2x na semana)

## Benefícios

- Plano de saúde e Assistência Odontológica;
- Vale Refeição/Alimentação no cartão Flash;
- Auxílio Híbrido;
- Gympass / Wellhub;
- Seguro de vida;
- Licença Maternidade/Paternidade estendida;
- Parcerias e convênios diversos em educação, saúde e lazer (Universidades, escolas de Idioma, academias, clínicas de saúde...);

...
#### Obrigatórios:

· Graduação em Ciência da Computação ou áreas afins;
· Google Cloud;
· Big query;
· Linguagem de programação Python;
· MLOPS (de maneira geral);
· Estatística clássica
· Data pipelines;
· Docker (Conteiners).

#### Desejáveis:

· Vertex e cortex;
· Keras/tensorflow;
· Pytorch;
· Scikit;
· Xgboosd;
· Kedro;
· Streamlit.

## Contratação

- CLT

## Nossa empresa

Nós somos a BHS!
Com 31 anos de experiência no mercado, proporcionamos soluções modernas de TI. 🚀

Somos reconhecidos como a MELHOR empresa para se trabalhar em MG e TI do BRASIL pelo GPTW - Great Place to Work, e estamos classificados como a 4ª melhor empresa para se trabalhar na categoria LATAM.

## Como se candidatar

- https://bhs.gupy.io/jobs/11562751?jobBoardSource=gupy_public_page

## Tempo médio de feedbacks

Costumamos enviar feedbacks em até 10 dias após cada processo.

#### Alocação
- Alocado
- Hibrido

#### Regime
- CLT

#### Nível

- Sênior
